sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -19,8 +19,7 @@ internal class RequestActivityPolicy : HttpPipelinePolicy
         private const string RequestIdHeaderName = "Request-Id";
 
         private static readonly DiagnosticListener s_diagnosticSource = new DiagnosticListener("Azure.Core");
-#if NETCOREAPP2_1
-        private static readonly object? s_activitySource = ActivityExtensions.CreateActivitySource("Azure.Core.Http");
+#if NETCOREAPP2_1 // Tracing is disabled in netcoreapp2.1
```

Comment:
nit: Could we simplify this to:
```c#
#if !NETCOREAPP2_1 // Only enable tracing if not netcoreapp2.1, since tracing is disabled in netcoreapp2.1
    private static readonly ActivitySource s_activitySource = new ActivitySource("Azure.Core.Http");
#endif

if so, same for the others.
```